content/en/news/releases/1.12.x/announcing-1.12.2/index.md

```diff
@@ -9,7 +9,7 @@ aliases:
     - /news/announcing-1.12.2
 ---
 
-This release fixes security vulnerabilities described on January 18th ([ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001) and [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002)) and includes minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
```

Comment:
this is a load bearing bracket, it should render as 


> This release fixes security vulnerabilities described on January 18th (ISTIO-SECURITY-2022-001 and ISTIO-SECURITY-2022-002) and includes minor bug fixes to improve robustness. 

Was it breaking the lint?

Reply:
No, I just screwed it up - should be good now. thanks for the pointer